tests/models/test_validation.py

```diff
@@ -191,6 +191,54 @@ def test_validation_faulty_baseline_model(
             )
 
 
+@pytest.fixture
+def faulty_validation_thresholds_param_test_spec(request):
+    """
+    Test specification for faulty `baseline_model` parameter tests:
+    :return: validation_thresholds: A dictonary mapping scalar metric names
+             to MetricThreshold(threshold=0.2, higher_is_better=True),
+    """
+    if request.param == "param_not_dict":
+        return 1
+    if request.param == "key_not_str":
+        return {1: MetricThreshold(min_absolute_change=0.1, higher_is_better=True)}
+    if request.param == "value_not_metric_threshold":
+        return {"accuracy": 1}
+
+
+@pytest.mark.parametrize(
+    "faulty_validation_thresholds_param_test_spec",
+    [
+        ("param_not_dict"),
+        ("key_not_str"),
+        ("value_not_metric_threshold"),
+    ],
+    indirect=["faulty_validation_thresholds_param_test_spec"],
+)
```

Comment:
```suggestion
@pytest.mark.parametrize(
    "validation_thresholds",
    [
        pytest.param(1, id="param_not_dict"),
        pytest.param(
            {1: MetricThreshold(min_absolute_change=0.1, higher_is_better=True)}, id="key_not_str"
        ),
        pytest.param({"accuracy": 1}, id="value_not_metric_threshold"),
    ],
)
```

To name parameters, we can also use `pytest.param` with `id`.